src/styles/blocks/nav.scss

```
@@ -0,0 +1,52 @@
+.nav {
+
+  &__list {
```

remove redundant spaces 
```suggestion
.nav {
  &__list {
```